features/samples.feature

```diff
@@ -50,6 +50,7 @@ Feature: Basic smell detection
       Inline::C#parse_signature has the variable name 'x' (UncommunicativeVariableName)
       Inline::C#parse_signature is controlled by argument raw (ControlParameter)
       Inline::C#strip_comments doesn't depend on instance state (UtilityFunction)
+      Module has no descriptive comment (IrresponsibleModule)
```

Comment:
Same here. This should have failed before but it didnt since "class Module" indeed does not have a descriptive comment.